colibre/auto_plotter/gas_fraction_plots.yml

```diff
@@ -72,10 +72,10 @@ h2_frac_func_stellar_surface_density:
   x:
     quantity: "derived_quantities.mu_star_100_kpc"
     units: "Solar_Mass / (kpc**2)"
-    start: 1e9
+    start: 1e7
```

Comment:
You should change the number of bins here too, to make sure it's all consistent with the standard 0.2 dex.

Reply:
I've set all to 0.2 dex by default - but i think this is probably too narrow for runs particularly in small boxes as any high dispersion plots can yield very noisy trendlines

Reply:
That's why we use the adaptive lines, so hopefully we shouldn't have a problem.